### PR TITLE
Rework the uncss internals to use postcss.

### LINKED ISSUE #153: Switch to PostCSS
/CC @giakki @ben-eb

### PR DESCRIPTION
**Pull request by mikelambert**
I believe this should maintain API compatibility, though the error-handling will need to change.

This should be a precursor step to sourcemap support, as part of writing a postcss plugin on top of the reworked lib.js.

Fixes #153.

### CODE REVIEW

**Comment by mikelambert** [comment]
Just a headsup on the test changes, postcss appears to preserve whitespace with the original files, which means running it through uncss will no longer normalize minified or unlinted CSS. This has resulted in a bunch of test data changes (I'm open to other solutions here if this bugs you, though).

**Comment by XhmikosR** [comment]
Great job @mikelambert! I'll wait for @giakki to review this.

**Comment by XhmikosR** [comment]
@giakki: this should be merged first since #214 depends on it.

File: package.json [hunk 1]
```diff
@@ -35,12 +35,12 @@
   "dependencies": {
     "bluebird": "~3.1.5",
     "commander": "~2.9.0",
-    "css": "1.6.0",
     "glob": "~6.0.1",
     "is-absolute-url": "~1.0.0",
     "is-html": "~1.0.0",
     "lodash": "~4.0.1",
     "phridge": "~1.2.2",
+    "postcss": "^5.0.14",
```
Comment (XhmikosR) [inline package.json]: Can you use tilde for consistency?

**Comment by mikelambert** [comment]
This one is super-ceded as part of merging #214 , so will just close this one without merging.

**Pull request closed without merging**